trouble_shooting.md

```diff
@@ -0,0 +1,12 @@
+# AKS Troubleshooting guide
+
+## Problems
+
+### Exposed service is not reachable from Internet
+
+When the exposed service is not reacheable, the first thing to verify is whether the NSG rules are modified outs
```

Comment:
I would mention that there are three components. You would want to check the LB,NSG and potentially kube-proxy. I would also put a reference to the describe commands via kubectl so that you can confirm the sync loops on the cloud-controller code isn't throwing errors before checking the Azure side of things

Reply:
Pinging this again